conf/modules/guidance_indi_hybrid.xml

```diff
@@ -9,7 +9,7 @@
   <settings>
     <dl_settings>
       <dl_settings NAME="guidance_indi_hybrid">
-        <dl_setting var="gih_params.liftd_p50" min="0.1" step="0.1" max="10.0" shortname="liftd_p50" param="GUIDANCE_INDI_LIFTD_P50" persistent="true"/>
+        <dl_setting var="gih_params.liftd_p50" min="0.1" step="0.1" max="10.0" shortname="liftd_p50" param="GUIDANCE_INDI_LIFTD_P50" persistent="true" module="guidance/guidance_indi_hybrid"/>
```

Comment:
Why is there a module added here?

Reply:
The `module` attribute is a legacy prior to the "modules" we have now. It can be replaced by `header` and it was used to specified the header file for settings.
We will clean that someday...